### PR TITLE
Readme submission

### PR DESCRIPTION
**Pull request by manojdvs7**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
-# **RND2020**
+# **RnD-2020**
 ## **Learning**
```
Comment (krushiraj) [inline README.md:2]:
```
# - is equivalent to h1
## - is equivalent to h2
```
I need you to find something which is equivalent to ul, ol and li in md. Please make the changes for line no `3` after you find those.

**Review by krushiraj — CHANGES_REQUESTED** [review]
Please use the suggested equivalents for list and update.

File: README.md (hunk 1 above)
Comment (manojdvs7) [reply README.md:2]: should learning also be a list?

Comment (krushiraj) [reply README.md:2]: Nope. Learning should be like side heading for the list below. From now, you'll be adding things to list items periodically based on the lessons and assignments done.

Comment (manojdvs7) [reply README.md:2]: bro,
I found equivalent from this link https://www.markdownguide.org/basic-syntax/#unordered-lists .What changes should I make in line 3?

Comment (krushiraj) [reply README.md:2]: Sorry, my bad I thought there was another `*` at the end of line

It's okay if you use `*` or `-` for list item. 

Will merge this now.

**Review by krushiraj — APPROVED** [review]
Added README

File: README.md (hunk 1 above)
Comment (manojdvs7) [reply README.md:2]: No problem bro.Thanks

**Pull request merged**